test/Datadog.Trace.Tools.Analyzers.Tests/ThreadAbortAnalyzerTests.cs

```diff
@@ -0,0 +1,208 @@
+﻿// <copyright file="ThreadAbortAnalyzerTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using VerifyCS = Datadog.Trace.Tools.Analyzers.Tests.CSharpCodeFixVerifier<
+    Datadog.Trace.Tools.Analyzers.ThreadAbortAnalyzer.ThreadAbortAnalyzer,
+    Datadog.Trace.Tools.Analyzers.ThreadAbortAnalyzer.ThreadAbortCodeFixProvider>;
+
+namespace Datadog.Trace.Tools.Analyzers.Tests
+{
+    [TestClass]
+    public class ThreadAbortAnalyzerTests
+    {
+        private const string DiagnosticId = ThreadAbortAnalyzer.ThreadAbortAnalyzer.DiagnosticId;
+
+        // No diagnostics expected to show up
+        [TestMethod]
```

Comment:
Would you mind updating these attributes to `[Fact]`, `[Theory]`, etc.?

Reply:
Oops, my bad! Fixed in https://github.com/DataDog/dd-trace-dotnet/pull/1325/commits/c6bade60b87a4c4cc4066238b4adaf5d07eb5875